lib/tasks/preflight.ex

```diff
@@ -18,4 +23,17 @@ defmodule Mix.Tasks.Preflight do
       System.halt(:abort)
     end
   end
+
+  def check_hex_version do
+    required_version_num = "0.16.0"
+
+    hex_info = "mix hex.info | grep Hex" |> String.to_char_list |> :os.cmd |> List.to_string
```

Comment:
@vanderhoop Thanks!  Still new to elixir, so I'm not always sure what the best way to do things is.